src/pages/_DevSitePageErrorBoundary.jsx

```diff
@@ -1,17 +1,14 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import ErrorPage from './_ErrorPage';
+import ApplicationStatusOverlay from '@cerner/terra-application/lib/application-status-overlay';
+
+import ContentLoadedContainer from './_ContentLoaded';
 
 const propTypes = {
   /**
    * children are children
```

Comment:
Profound :) Can we update this description?

Reply:
Recommendations? I never know what to say about 'children'.